<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Prathyush-PC" ignored="8" name="Functional Test" tests="9" failures="6" timestamp="30 Jan 2019 14:45:57 GMT" time="27.421" errors="0">
  <testcase name="LGN_001" time="18.358" classname="com.wibmo.LoginAPI.tests.LGN_001"/>
  <testcase name="LGN_002" time="0.968" classname="com.wibmo.LoginAPI.tests.LGN_002">
    <failure type="java.lang.AssertionError" message="expected [BAD_PWD] but found [BLOCKED]">
      <![CDATA[java.lang.AssertionError: expected [BAD_PWD] but found [BLOCKED]
	at com.wibmo.LoginAPI.tests.LGN_002.LGN_002(LGN_002.java:39)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- LGN_002 -->
  <testcase name="LGN_003" time="0.844" classname="com.wibmo.LoginAPI.tests.LGN_003"/>
  <testcase name="LGN_004" time="0.657" classname="com.wibmo.LoginAPI.tests.LGN_004"/>
  <testcase name="IAPV2_001" time="2.405" classname="com.wibmo.IAPV2.tests.IAPV2_001">
    <failure type="java.lang.AssertionError" message="expected [50020] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [50020] but found [null]
	at com.wibmo.IAPV2.tests.IAPV2_001.IAPV2_001(IAPV2_001.java:46)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- IAPV2_001 -->
  <testcase name="IAPV2_002" time="0.75" classname="com.wibmo.IAPV2.tests.IAPV2_002">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:930)
	at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:895)
	at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:975)
	at io.restassured.path.json.JsonPath.get(JsonPath.java:201)
	at io.restassured.path.json.JsonPath.getString(JsonPath.java:351)
	at com.wibmo.IAPV2.tests.IAPV2_002.IAPV2_002(IAPV2_002.java:39)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
	at groovy.json.JsonLexer.nextToken(JsonLexer.java:83)
	at groovy.json.JsonLexer$nextToken.call(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
	at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:893)
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)
	... 30 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- IAPV2_002 -->
  <testcase name="IAPV2_003" time="0.406" classname="com.wibmo.IAPV2.tests.IAPV2_003">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:930)
	at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:895)
	at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:975)
	at io.restassured.path.json.JsonPath.get(JsonPath.java:201)
	at library.Generic.getJsonPathValue(Generic.java:27)
	at com.wibmo.API.LoginAPI.Login(LoginAPI.java:53)
	at com.wibmo.IAPV2.tests.IAPV2_003.IAPV2_003(IAPV2_003.java:35)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
	at groovy.json.JsonLexer.nextToken(JsonLexer.java:83)
	at groovy.json.JsonLexer$nextToken.call(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
	at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:893)
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)
	... 31 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- IAPV2_003 -->
  <testcase name="LM_001" time="0.429" classname="com.wibmo.LoadMoneyIAP.tests.LM_001">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:930)
	at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:895)
	at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:975)
	at io.restassured.path.json.JsonPath.get(JsonPath.java:201)
	at library.Generic.getJsonPathValue(Generic.java:27)
	at com.wibmo.API.LoginAPI.Login(LoginAPI.java:53)
	at com.wibmo.LoadMoneyIAP.tests.LM_001.LM_001(LM_001.java:34)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
	at groovy.json.JsonLexer.nextToken(JsonLexer.java:83)
	at groovy.json.JsonLexer$nextToken.call(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
	at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:893)
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)
	... 31 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- LM_001 -->
  <testcase name="SM_001" time="1.342" classname="com.wibmo.SendMoneyAPI.tests.SM_001">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:930)
	at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:895)
	at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:975)
	at io.restassured.path.json.JsonPath.get(JsonPath.java:201)
	at library.Generic.getJsonPathValue(Generic.java:27)
	at com.wibmo.SendMoneyAPI.tests.SM_001.SM_001(SM_001.java:39)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
	at groovy.json.JsonLexer.nextToken(JsonLexer.java:83)
	at groovy.json.JsonLexer$nextToken.call(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
	at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
	at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:893)
	at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:928)
	... 30 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- SM_001 -->
  <testcase name="@BeforeSuite initializeReports" time="0.0" classname="library.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite initializeReports -->
  <testcase name="@BeforeSuite initializeReports" time="0.0" classname="library.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite initializeReports -->
  <testcase name="@BeforeSuite initializeReports" time="0.0" classname="library.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite initializeReports -->
  <testcase name="@BeforeSuite initializeReports" time="0.0" classname="library.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite initializeReports -->
  <testcase name="@BeforeSuite initializeReports" time="0.0" classname="library.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite initializeReports -->
  <testcase name="@BeforeSuite initializeReports" time="0.0" classname="library.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite initializeReports -->
  <testcase name="@BeforeSuite initializeReports" time="0.0" classname="library.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite initializeReports -->
  <testcase name="@BeforeSuite initializeReports" time="0.0" classname="library.BaseClass">
    <ignored/>
  </testcase> <!-- @BeforeSuite initializeReports -->
</testsuite> <!-- Functional Test -->
